tools/benchmark/pnpm-lock.yaml

```diff
@@ -356,7 +353,7 @@ packages:
   /@rushstack/node-core-library@5.3.0(@types/node@14.18.38):
     resolution: {integrity: sha512-t23gjdZV6aWkbwXSE3TkKr1UXJFbXICvAOJ0MRQEB/ZYGhfSJqqrQFaGd20I1a/nIIHJEkNO0xzycHixjcbCPw==}
     peerDependencies:
-      '@types/node': ^18.19.0
+      '@types/node': '*'
```

Comment:
This bit feels weird and unrelated to the removal, probably revert? Same on line 385.